src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -549,7 +549,494 @@ type Address @model {
 ```
 
 ### Relational models with custom primary keys
-TODO
+You can define custom primary keys for relational models. The transformer will handle the foreign key fields needed by queries and mutations, which allows you to retrieve data across different relations. 
+
+You can choose the default relations with **implicit** fields. This will result in auto generation of additional foreign key fields refering its counterpart primary key and sort key fields.
```

Comment:
Similar here "implicit fields" is a new term in the documentation. 
Would be better to use existing terms
e.g.

The fields arguments of `@belongsTo` directive.
`@belongTo` can be used without the `fields` argument. (this is the implicit I believe)

Reply:
I am using `implicit` and `explicit` terms in the block header. So I add additional explanations on these two words